frontend/packages/ux-editor/src/hooks/mutations/useAddGroupMutation.ts

```
@@ -0,0 +1,58 @@
+  const { t } = useTranslation();
+
+  return useMutation({
+    mutationFn: async () => {
+      try {
+        const updatedPages = await getPages(org, app, selectedFormLayoutSetName);
+        const nextPageNumber = getNextPageNumber(updatedPages.groups, t);
+        const newGroup = createNewGroup(updatedPages.groups, nextPageNumber, t);
+        const finalPayload = addGroupsWithPages(updatedPages, newGroup);
+        return await changePageGroups(org, app, selectedFormLayoutSetName, finalPayload);
+      } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)